packages/variants/src/variantThemeType.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * Const enum for variant theme types.
+ *
+ * @public
+ */
+
+export const enum VariantThemeType {
+  None = 0,
+  Neutral = 1,
```

Comment:
Would be great to jsdoc these enums too (which is the JSDoc minbar in office-ui-fabric-react)